test/models/ocelot/stanzas/InputStanzaSpec.scala

```diff
@@ -66,264 +66,162 @@ class InputStanzaSpec extends BaseSpec {
 
   "CurrencyInput " should {
     "update the input label" in {
-
-      Input(expectedCurrencyStanza, Phrase("",""), None, None).get match {
-        case currencyInput: CurrencyInput =>
-          val labels = LabelCache()
-
-          val (nxt, updatedLabels) = currencyInput.eval("33", labels)
-          updatedLabels.updatedLabels(expectedCurrencyStanza.label).english shouldBe Some("33")
-        case _ => fail
-      }
-
+      val input = Input(expectedCurrencyStanza, Phrase("",""), None, None)
+      val labels = LabelCache()
+      val (nxt, updatedLabels) = input.eval("33", labels)
```

Comment:
Intellij warning - nxt not used, replace with _

Reply:
done